### ISSUE TITLE
[A11y_Bug]: The Rule IDs in accessibility are changed/updated for which we are unable to map Rule ids with num, name, wcagLevel etc. 

### ISSUE THREAD

**Issue by SaiManasal** [IBMa/equal-access#1482]
### Project

a11y checker

### Browser

Chrome

### Operating System

Windows

### Automated testing tool and ruleset

IBM Accessibility Checker - Latest Deployment

### Description

Due to change in Rule Id the mapping to WCAG rule is distributed results to change in output results.
Can you please provide the mapping for latest updated RuleID with WCAG .
 Because of change in accessibility ruleids we are unable to calculate the complete score , fail rules, need review rules etc. and are not providing accurate  rule results as expected. .
If the rule IDs are updated in accessibility checker then ruleset should also contain all its respective rule ids, num, name, wcagLevel etc. 

### Steps to reproduce

To reproduce the issue please trigger the IBM accessibility with npm achecker command for which you will be receiving the response with updated rules id. In ruleset as well we are not be able to check the updated rules ids and its mapped WCAG rules,
